strawberryfields/gbs/vibronic.py

```diff
@@ -194,26 +194,28 @@ def gbs_params(
     return t, U1, np.log(s), U2, alpha
 
 
-def energies(samples: list, wp: np.ndarray) -> Union[list, float]:
+def energies(samples: list, w: np.ndarray, wp: np.ndarray) -> Union[list, float]:
     r"""Computes the energy of each GBS sample in units of :math:`\text{cm}^{-1}`.
 
```

Comment:
We should probably include here a line stating how the energy is defined in terms of the sample, w, and wp

Reply:
It will be explained, with proper equations, in the docstring.